web/documentserver-example/nodejs/app.js

```diff
@@ -651,6 +651,22 @@ app.get("/editor", function (req, res) {
     }
 });
 
+app.get("/files", function(req, res){
```

Comment:
Who will use this request? Why add it?

Reply:
I dont know. I didn't know how to correctly implement the logic.